floss/const.py

```diff
@@ -3,7 +3,7 @@
 KILOBYTE = 1024
 MEGABYTE = 1024 * KILOBYTE
 MAX_FILE_SIZE = 16 * MEGABYTE
-SUPPORTED_FILE_MAGIC = {b"MZ"}
+SUPPORTED_FILE_MAGIC = {b"MZ",b"\x7fELF"}
```

Comment:
```suggestion
SUPPORTED_FILE_MAGIC = {b"MZ", b"\x7fELF"}
```

Reply:
see https://github.com/mandiant/flare-floss/pull/969 where tests are enabled for this branch